src/operations.jl

```diff
@@ -62,3 +62,23 @@ end
 function decrypt(secure_matrix::SecureMatrix, private_key::PrivateKey)
     decrypt_impl(secure_matrix, private_key)
 end
+
+"""
+    cleanup()
```

Comment:
I suggest to use a more descriptive name. Maybe something like `clear_contexts` or `release_context_memory`?

Reply:
`release_context_memory` sounds well!